lib/ae_mdw/txs.ex

```diff
@@ -209,6 +211,22 @@ defmodule AeMdw.Txs do
     tx_hash
   end
 
+  @spec fetch_pending_txs(state(), pagination(), range(), cursor()) ::
+          {:ok, {page_cursor(), [tx()], page_cursor()}} | {:error, Error.t()}
```

Comment:
Looks like there's no possible `{:error, Error.t()}` flow

Reply:
You're correct, there was in an earlier implementation that could. Removing it